[libidn2] Build failue

Package: libidn2[core,nls]:x64-osx -> 2.3.4#2

**Host Environment**

- Host: x64-osx
- Compiler: AppleClang 14.0.3.14030022
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: 03a211eb3 2023-04-20 (14 hours ago)

**To Reproduce**

`vcpkg install --enforce-port-checks --allow-unsupported`

**Failure logs**

```
-- Using cached libidn2-2.3.4.tar.gz.
-- Cleaning sources at /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libidn2/src/v2.3.4-5cd585afd5.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /Users/leanderSchulten/git_projekte/vcpkg/downloads/libidn2-2.3.4.tar.gz
-- Applying patch fix-static-into-shared-linking.patch
-- Applying patch disable-subdirs.patch
-- Applying patch fix-msvc.patch
-- Applying patch fix-uwp.patch
-- Using source at /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libidn2/src/v2.3.4-5cd585afd5.clean
-- Found external ninja('1.11.1').
-- Getting CMake variables for x64-osx-dbg
-- Getting CMake variables for x64-osx-rel
-- Generating configure for x64-osx
-- Finished generating configure for x64-osx
-- Configuring x64-osx-dbg
-- Configuring x64-osx-rel
-- Building x64-osx-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /usr/bin/make V=1 -j 11 -f Makefile all
    Working Directory: /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libidn2/x64-osx-dbg/
    See logs for more information:
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libidn2/build-x64-osx-dbg-out.log
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libidn2/build-x64-osx-dbg-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_make.cmake:151 (vcpkg_execute_build_process)
  scripts/cmake/vcpkg_install_make.cmake:2 (vcpkg_build_make)
  ports/libidn2/portfile.cmake:60 (vcpkg_install_make)
  scripts/ports.cmake:147 (include)



```
<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libidn2/build-x64-osx-dbg-out.log</summary>

```
/Applications/Xcode.app/Contents/Developer/usr/bin/make  all-recursive
Making all in gl
sed -e 1h -e '1s,.*,/* DO NOT EDIT! GENERATED AUTOMATICALLY! */,' -e 1G \
	  -e 's|@''HAVE_ALLOCA_H''@|1|g' \
	  ./alloca.in.h > alloca.h-t
{ sed -e 1h -e '1s,.*,/* DO NOT EDIT! GENERATED AUTOMATICALLY! */,' -e 1G \
	      -e 's|@''INCLUDE_NEXT''@|include_next|g' \
	      -e 's|@''PRAGMA_SYSTEM_HEADER''@|#pragma GCC system_header|g' \
	      -e 's|@''PRAGMA_COLUMNS''@||g' \
	      -e 's|@''NEXT_ASSERT_H''@|<assert.h>|g' \
	      < ./assert.in.h && \
	  sed -e '/@assert.h omit start@/,/@assert.h omit end@/d' \
	      -e 's|_gl_verify|_gl_static_assert|g' \
	      -e 's|_GL_VERIFY|_GL_STATIC_ASSERT|g' \
	      -e 's|_GL\(_STATIC_ASSERT_H\)|_GL\1|g' \
	      < ./verify.h; \
	} > assert.h-t
sed -e 1h -e '1s,.*,/* DO NOT EDIT! GENERATED AUTOMATICALLY! */,' -e 1G \
	      -e 's|@''GUARD_PREFIX''@|GL|g' \
	      -e 's|@''INCLUDE_NEXT''@|include_next|g' \
	      -e 's|@''PRAGMA_SYSTEM_HEADER''@|#pragma GCC system_header|g' \
	      -e 's|@''PRAGMA_COLUMNS''@||g' \
	      -e 's|@''NEXT_FCNTL_H''@|<fcntl.h>|g' \
	      -e 's/@''GNULIB_CREAT''@/0/g' \
	      -e 's/@''GNULIB_FCNTL''@/1/g' \
	      -e 's/@''GNULIB_NONBLOCKING''@/0/g' \
	      -e 's/@''GNULIB_OPEN''@/1/g' \
	      -e 's/@''GNULIB_OPENAT''@/0/g' \
	      -e 's/@''GNULIB_MDA_CREAT''@/1/g' \
	      -e 's/@''GNULIB_MDA_OPEN''@/1/g' \
	      -e 's|@''HAVE_FCNTL''@|1|g' \
	      -e 's|@''HAVE_OPENAT''@|1|g' \
	      -e 's|@''REPLACE_CREAT''@|0|g' \
	      -e 's|@''REPLACE_FCNTL''@|0|g' \
	      -e 's|@''REPLACE_OPEN''@|1|g' \
	      -e 's|@''REPLACE_OPENAT''@|0|g' \
	      -e '/definitions of _GL_FUNCDECL_RPL/r ./c++defs.h' \
	      -e '/definition of _GL_ARG_NONNULL/r ./arg-nonnull.h' \
	      -e '/definition of _GL_WARN_ON_USE/r ./warn-on-use.h' \
	      ./fcntl.in.h > fcntl.h-t
sed -e 1h -e '1s,.*,/* DO NOT EDIT! GENERATED AUTOMATICALLY! */,' -e 1G \
	      -e 's|@''GUARD_PREFIX''@|GL|g' \
	      -e 's|@''HAVE_GETOPT_H''@|1|g' \
	      -e 's|@''INCLUDE_NEXT''@|include_next|g' \
	      -e 's|@''PRAGMA_SYSTEM_HEADER''@|#pragma GCC system_header|g' \
	      -e 's|@''PRAGMA_COLUMNS''@||g' \
	      -e 's|@''NEXT_GETOPT_H''@|<getopt.h>|g' \
	      -e '/definition of _GL_ARG_NONNULL/r ./arg-nonnull.h' \
	      ./getopt.in.h > getopt.h-t
sed -e 1h -e '1s,.*,/* DO NOT EDIT! GENERATED AUTOMATICALLY! */,' -e 1G \
	  -e 's|@''HAVE_SYS_CDEFS_H''@|1|g' \
	  ./getopt-cdefs.in.h > getopt-cdefs.h-t
sed -e 1h -e '1s,.*,/* DO NOT EDIT! GENERATED AUTOMATICALLY! */,' -e 1G \
	      -e 's|@''GUARD_PREFIX''@|GL|g' \
	      -e 's|@''INCLUDE_NEXT''@|include_next|g' \
	      -e 's|@''PRAGMA_SYSTEM_HEADER''@|#pragma GCC system_header|g' \
	      -e 's|@''PRAGMA_COLUMNS''@||g' \
	      -e 's|@''NEXT_LIMITS_H''@|<limits.h>|g' \
	      ./limits.in.h > limits.h-t
sed -e 1h -e '1s,.*,/* DO NOT EDIT! GENERATED AUTOMATICALLY! */,' -e 1G \
	  ./stdckdint.in.h > stdckdint.h-t
sed -e 1h -e '1s,.*,/* DO NOT EDIT! GENERATED AUTOMATICALLY! */,' -e 1G \
...
Skipped 146 lines
...

	      -e '/definitions of _GL_FUNCDECL_RPL/r ./c++defs.h' \
	      -e '/definition of _GL_ARG_NONNULL/r ./arg-nonnull.h' \
	      -e '/definition of _GL_WARN_ON_USE/r ./warn-on-use.h' \
	      > stdio.h-t
sed -e 1h -e '1s,.*,/* DO NOT EDIT! GENERATED AUTOMATICALLY! */,' -e 1G \
	      -e 's|@''GUARD_PREFIX''@|GL|g' \
	      -e 's|@''INCLUDE_NEXT''@|include_next|g' \
	      -e 's|@''PRAGMA_SYSTEM_HEADER''@|#pragma GCC system_header|g' \
	      -e 's|@''PRAGMA_COLUMNS''@||g' \
	      -e 's|@''NEXT_STDLIB_H''@|<stdlib.h>|g' \
	      -e 's/@''GNULIB__EXIT''@/0/g' \
	      -e 's/@''GNULIB_ALIGNED_ALLOC''@/0/g' \
	      -e 's/@''GNULIB_ATOLL''@/0/g' \
	      -e 's/@''GNULIB_CALLOC_GNU''@/0/g' \
	      -e 's/@''GNULIB_CALLOC_POSIX''@/0/g' \
	      -e 's/@''GNULIB_CANONICALIZE_FILE_NAME''@/0/g' \
	      -e 's/@''GNULIB_FREE_POSIX''@/0/g' \
	      -e 's/@''GNULIB_GETLOADAVG''@/0/g' \
	      -e 's/@''GNULIB_GETSUBOPT''@/0/g' \
	      -e 's/@''GNULIB_GRANTPT''@/0/g' \
	      -e 's/@''GNULIB_MALLOC_GNU''@/0/g' \
	      -e 's/@''GNULIB_MALLOC_POSIX''@/0/g' \
	      -e 's/@''GNULIB_MBTOWC''@/0/g' \
	      -e 's/@''GNULIB_MKDTEMP''@/0/g' \
	      -e 's/@''GNULIB_MKOSTEMP''@/0/g' \
	      -e 's/@''GNULIB_MKOSTEMPS''@/0/g' \
	      -e 's/@''GNULIB_MKSTEMP''@/0/g' \
	      -e 's/@''GNULIB_MKSTEMPS''@/0/g' \
	      -e 's/@''GNULIB_POSIX_MEMALIGN''@/0/g' \
	      -e 's/@''GNULIB_POSIX_OPENPT''@/0/g' \
	      -e 's/@''GNULIB_PTSNAME''@/0/g' \
	      -e 's/@''GNULIB_PTSNAME_R''@/0/g' \
	      -e 's/@''GNULIB_PUTENV''@/0/g' \
	      -e 's/@''GNULIB_QSORT_R''@/0/g' \
	      -e 's/@''GNULIB_RANDOM''@/0/g' \
	      -e 's/@''GNULIB_RANDOM_R''@/0/g' \
	      -e 's/@''GNULIB_REALLOC_GNU''@/0/g' \
	      -e 's/@''GNULIB_REALLOC_POSIX''@/0/g' \
	      -e 's/@''GNULIB_REALLOCARRAY''@/0/g' \
	      -e 's/@''GNULIB_REALPATH''@/0/g' \
	      -e 's/@''GNULIB_RPMATCH''@/0/g' \
	      -e 's/@''GNULIB_SECURE_GETENV''@/0/g' \
	      -e 's/@''GNULIB_SETENV''@/0/g' \
	      -e 's/@''GNULIB_STRTOD''@/0/g' \
	      -e 's/@''GNULIB_STRTOL''@/0/g' \
	      -e 's/@''GNULIB_STRTOLD''@/0/g' \
	      -e 's/@''GNULIB_STRTOLL''@/0/g' \
	      -e 's/@''GNULIB_STRTOUL''@/0/g' \
	      -e 's/@''GNULIB_STRTOULL''@/0/g' \
	      -e 's/@''GNULIB_SYSTEM_POSIX''@/0/g' \
	      -e 's/@''GNULIB_UNLOCKPT''@/0/g' \
	      -e 's/@''GNULIB_UNSETENV''@/0/g' \
	      -e 's/@''GNULIB_WCTOMB''@/0/g' \
	      -e 's/@''GNULIB_MDA_ECVT''@/1/g' \
	      -e 's/@''GNULIB_MDA_FCVT''@/1/g' \
	      -e 's/@''GNULIB_MDA_GCVT''@/1/g' \
	      -e 's/@''GNULIB_MDA_MKTEMP''@/1/g' \
	      -e 's/@''GNULIB_MDA_PUTENV''@/1/g' \
	      < ./stdlib.in.h | \
	  sed -e 's|@''HAVE__EXIT''@|1|g' \
	      -e 's|@''HAVE_ALIGNED_ALLOC''@|1|g' \
	      -e 's|@''HAVE_ATOLL''@|1|g' \
	      -e 's|@''HAVE_CANONICALIZE_FILE_NAME''@|1|g' \
	      -e 's|@''HAVE_DECL_ECVT''@|1|g' \
	      -e 's|@''HAVE_DECL_FCVT''@|1|g' \
	      -e 's|@''HAVE_DECL_GCVT''@|1|g' \
	      -e 's|@''HAVE_DECL_GETLOADAVG''@|1|g' \
	      -e 's|@''HAVE_GETSUBOPT''@|1|g' \
	      -e 's|@''HAVE_GRANTPT''@|1|g' \
	      -e 's|@''HAVE_INITSTATE''@|1|g' \
	      -e 's|@''HAVE_DECL_INITSTATE''@|1|g' \
	      -e 's|@''HAVE_MBTOWC''@|1|g' \
	      -e 's|@''HAVE_MKDTEMP''@|1|g' \
	      -e 's|@''HAVE_MKOSTEMP''@|1|g' \
	      -e 's|@''HAVE_MKOSTEMPS''@|1|g' \
	      -e 's|@''HAVE_MKSTEMP''@|1|g' \
	      -e 's|@''HAVE_MKSTEMPS''@|1|g' \
	      -e 's|@''HAVE_POSIX_MEMALIGN''@|1|g' \
	      -e 's|@''HAVE_POSIX_OPENPT''@|1|g' \
	      -e 's|@''HAVE_PTSNAME''@|1|g' \
	      -e 's|@''HAVE_PTSNAME_R''@|1|g' \
	      -e 's|@''HAVE_QSORT_R''@|1|g' \
	      -e 's|@''HAVE_RANDOM''@|1|g' \
	      -e 's|@''HAVE_RANDOM_H''@|1|g' \
	      -e 's|@''HAVE_RANDOM_R''@|1|g' \
	      -e 's|@''HAVE_REALLOCARRAY''@|1|g' \
	      -e 's|@''HAVE_REALPATH''@|1|g' \
	      -e 's|@''HAVE_RPMATCH''@|1|g' \
	      -e 's|@''HAVE_SECURE_GETENV''@|1|g' \
	      -e 's|@''HAVE_DECL_SETENV''@|1|g' \
	      -e 's|@''HAVE_SETSTATE''@|1|g' \
	      -e 's|@''HAVE_DECL_SETSTATE''@|1|g' \
	      -e 's|@''HAVE_STRTOD''@|1|g' \
	      -e 's|@''HAVE_STRTOL''@|1|g' \
	      -e 's|@''HAVE_STRTOLD''@|1|g' \
	      -e 's|@''HAVE_STRTOLL''@|1|g' \
	      -e 's|@''HAVE_STRTOUL''@|1|g' \
	      -e 's|@''HAVE_STRTOULL''@|1|g' \
	      -e 's|@''HAVE_STRUCT_RANDOM_DATA''@|1|g' \
	      -e 's|@''HAVE_SYS_LOADAVG_H''@|0|g' \
	      -e 's|@''HAVE_UNLOCKPT''@|1|g' \
	      -e 's|@''HAVE_DECL_UNSETENV''@|1|g' \
	      -e 's|@''REPLACE_ALIGNED_ALLOC''@|0|g' \
	      -e 's|@''REPLACE_CALLOC_FOR_CALLOC_GNU''@|0|g' \
	      -e 's|@''REPLACE_CALLOC_FOR_CALLOC_POSIX''@|0|g' \
	      -e 's|@''REPLACE_CANONICALIZE_FILE_NAME''@|0|g' \
	      -e 's|@''REPLACE_FREE''@|1|g' \
	      -e 's|@''REPLACE_INITSTATE''@|0|g' \
	      -e 's|@''REPLACE_MALLOC_FOR_MALLOC_GNU''@|0|g' \
	      -e 's|@''REPLACE_MALLOC_FOR_MALLOC_POSIX''@|0|g' \
	      -e 's|@''REPLACE_MBTOWC''@|0|g' \
	      -e 's|@''REPLACE_MKSTEMP''@|0|g' \
	      -e 's|@''REPLACE_POSIX_MEMALIGN''@|0|g' \
	      -e 's|@''REPLACE_PTSNAME''@|0|g' \
	      -e 's|@''REPLACE_PTSNAME_R''@|0|g' \
	      -e 's|@''REPLACE_PUTENV''@|0|g' \
	      -e 's|@''REPLACE_QSORT_R''@|0|g' \
	      -e 's|@''REPLACE_RANDOM''@|0|g' \
	      -e 's|@''REPLACE_RANDOM_R''@|0|g' \
	      -e 's|@''REPLACE_REALLOC_FOR_REALLOC_GNU''@|0|g' \
	      -e 's|@''REPLACE_REALLOC_FOR_REALLOC_POSIX''@|0|g' \
	      -e 's|@''REPLACE_REALLOCARRAY''@|0|g' \
	      -e 's|@''REPLACE_REALPATH''@|0|g' \
	      -e 's|@''REPLACE_SETENV''@|0|g' \
	      -e 's|@''REPLACE_SETSTATE''@|0|g' \
	      -e 's|@''REPLACE_STRTOD''@|0|g' \
	      -e 's|@''REPLACE_STRTOL''@|0|g' \
	      -e 's|@''REPLACE_STRTOLD''@|0|g' \
	      -e 's|@''REPLACE_STRTOLL''@|0|g' \
	      -e 's|@''REPLACE_STRTOUL''@|0|g' \
	      -e 's|@''REPLACE_STRTOULL''@|0|g' \
	      -e 's|@''REPLACE_UNSETENV''@|0|g' \
	      -e 's|@''REPLACE_WCTOMB''@|0|g' \
	      -e '/definitions of _GL_FUNCDECL_RPL/r ./c++defs.h' \
	      -e '/definition of _Noreturn/r ./_Noreturn.h' \
	      -e '/definition of _GL_ARG_NONNULL/r ./arg-nonnull.h' \
	      -e '/definition of _GL_WARN_ON_USE/r ./warn-on-use.h' \
	      > stdlib.h-t
sed -e 1h -e '1s,.*,/* DO NOT EDIT! GENERATED AUTOMATICALLY! */,' -e 1G \
	      -e 's|@''GUARD_PREFIX''@|GL|g' \
	      -e 's|@''INCLUDE_NEXT''@|include_next|g' \
	      -e 's|@''PRAGMA_SYSTEM_HEADER''@|#pragma GCC system_header|g' \
	      -e 's|@''PRAGMA_COLUMNS''@||g' \
	      -e 's|@''NEXT_STRING_H''@|<string.h>|g' \
	      -e 's/@''GNULIB_EXPLICIT_BZERO''@/0/g' \
	      -e 's/@''GNULIB_FFSL''@/0/g' \
	      -e 's/@''GNULIB_FFSLL''@/0/g' \
	      -e 's/@''GNULIB_MBSLEN''@/0/g' \
	      -e 's/@''GNULIB_MBSNLEN''@/0/g' \
	      -e 's/@''GNULIB_MBSCHR''@/0/g' \
	      -e 's/@''GNULIB_MBSRCHR''@/0/g' \
	      -e 's/@''GNULIB_MBSSTR''@/0/g' \
	      -e 's/@''GNULIB_MBSCASECMP''@/0/g' \
	      -e 's/@''GNULIB_MBSNCASECMP''@/0/g' \
	      -e 's/@''GNULIB_MBSPCASECMP''@/0/g' \
	      -e 's/@''GNULIB_MBSCASESTR''@/0/g' \
	      -e 's/@''GNULIB_MBSCSPN''@/0/g' \
	      -e 's/@''GNULIB_MBSPBRK''@/0/g' \
	      -e 's/@''GNULIB_MBSSPN''@/0/g' \
	      -e 's/@''GNULIB_MBSSEP''@/0/g' \
	      -e 's/@''GNULIB_MBSTOK_R''@/0/g' \
	      -e 's/@''GNULIB_MEMCHR''@/0/g' \
	      -e 's/@''GNULIB_MEMMEM''@/0/g' \
	      -e 's/@''GNULIB_MEMPCPY''@/0/g' \
	      -e 's/@''GNULIB_MEMRCHR''@/0/g' \
	      -e 's/@''GNULIB_RAWMEMCHR''@/1/g' \
	      -e 's/@''GNULIB_STPCPY''@/0/g' \
	      -e 's/@''GNULIB_STPNCPY''@/0/g' \
	      -e 's/@''GNULIB_STRCHRNUL''@/1/g' \
	      -e 's/@''GNULIB_STRDUP''@/0/g' \
	      -e 's/@''GNULIB_STRNCAT''@/0/g' \
	      -e 's/@''GNULIB_STRNDUP''@/0/g' \
	      -e 's/@''GNULIB_STRNLEN''@/0/g' \
	      -e 's/@''GNULIB_STRPBRK''@/0/g' \
	      -e 's/@''GNULIB_STRSEP''@/0/g' \
	      -e 's/@''GNULIB_STRSTR''@/0/g' \
	      -e 's/@''GNULIB_STRCASESTR''@/0/g' \
	      -e 's/@''GNULIB_STRTOK_R''@/0/g' \
	      -e 's/@''GNULIB_STRERROR''@/1/g' \
	      -e 's/@''GNULIB_STRERROR_R''@/0/g' \
	      -e 's/@''GNULIB_STRERRORNAME_NP''@/0/g' \
	      -e 's/@''GNULIB_SIGABBREV_NP''@/0/g' \
	      -e 's/@''GNULIB_SIGDESCR_NP''@/0/g' \
	      -e 's/@''GNULIB_STRSIGNAL''@/0/g' \
	      -e 's/@''GNULIB_STRVERSCMP''@/1/g' \
	      -e 's/@''GNULIB_MDA_MEMCCPY''@/1/g' \
	      -e 's/@''GNULIB_MDA_STRDUP''@/1/g' \
	      -e 's/@''GNULIB_FREE_POSIX''@/0/g' \
	      < ./string.in.h | \
	  sed -e 's|@''HAVE_EXPLICIT_BZERO''@|1|g' \
	      -e 's|@''HAVE_FFSL''@|1|g' \
	      -e 's|@''HAVE_FFSLL''@|1|g' \
	      -e 's|@''HAVE_MBSLEN''@|0|g' \
	      -e 's|@''HAVE_DECL_MEMMEM''@|1|g' \
	      -e 's|@''HAVE_MEMPCPY''@|1|g' \
	      -e 's|@''HAVE_DECL_MEMRCHR''@|1|g' \
	      -e 's|@''HAVE_RAWMEMCHR''@|0|g' \
	      -e 's|@''HAVE_STPCPY''@|1|g' \
	      -e 's|@''HAVE_STPNCPY''@|1|g' \
	      -e 's|@''HAVE_STRCHRNUL''@|0|g' \
	      -e 's|@''HAVE_DECL_STRDUP''@|1|g' \
	      -e 's|@''HAVE_DECL_STRNDUP''@|1|g' \
	      -e 's|@''HAVE_DECL_STRNLEN''@|1|g' \
	      -e 's|@''HAVE_STRPBRK''@|1|g' \
	      -e 's|@''HAVE_STRSEP''@|1|g' \
	      -e 's|@''HAVE_STRCASESTR''@|1|g' \
	      -e 's|@''HAVE_DECL_STRTOK_R''@|1|g' \
	      -e 's|@''HAVE_DECL_STRERROR_R''@|1|g' \
	      -e 's|@''HAVE_STRERRORNAME_NP''@|1|g' \
	      -e 's|@''HAVE_SIGABBREV_NP''@|1|g' \
	      -e 's|@''HAVE_SIGDESCR_NP''@|1|g' \
	      -e 's|@''HAVE_DECL_STRSIGNAL''@|1|g' \
	      -e 's|@''HAVE_STRVERSCMP''@|0|g' \
	      -e 's|@''REPLACE_FFSLL''@|0|g' \
	      -e 's|@''REPLACE_MEMCHR''@|0|g' \
	      -e 's|@''REPLACE_MEMMEM''@|0|g' \
	      -e 's|@''REPLACE_FREE''@|1|g' \
	      -e 's|@''REPLACE_STPNCPY''@|0|g' \
	      -e 's|@''REPLACE_STRCHRNUL''@|0|g' \
	      -e 's|@''REPLACE_STRDUP''@|0|g' \
	      -e 's|@''REPLACE_STRNCAT''@|0|g' \
	      -e 's|@''REPLACE_STRNDUP''@|0|g' \
	      -e 's|@''REPLACE_STRNLEN''@|0|g' \
	      -e 's|@''REPLACE_STRSTR''@|0|g' \
	      -e 's|@''REPLACE_STRCASESTR''@|0|g' \
	      -e 's|@''REPLACE_STRTOK_R''@|0|g' \
	      -e 's|@''REPLACE_STRERROR''@|1|g' \
	      -e 's|@''REPLACE_STRERROR_R''@|0|g' \
	      -e 's|@''REPLACE_STRERRORNAME_NP''@|0|g' \
	      -e 's|@''REPLACE_STRSIGNAL''@|0|g' \
	      -e 's|@''UNDEFINE_STRTOK_R''@|0|g' \
	      -e '/definitions of _GL_FUNCDECL_RPL/r ./c++defs.h' \
	      -e '/definition of _GL_ARG_NONNULL/r ./arg-nonnull.h' \
	      -e '/definition of _GL_WARN_ON_USE/r ./warn-on-use.h' \
	      > string.h-t
mv getopt-cdefs.h-t getopt-cdefs.h
mv limits.h-t limits.h
mv stdint.h-t stdint.h
mv getopt.h-t getopt.h
mv assert.h-t assert.h
mv stdlib.h-t stdlib.h
mv fcntl.h-t fcntl.h
mv stdckdint.h-t stdckdint.h
mv string.h-t string.h
mv stdio.h-t stdio.h
mv alloca.h-t alloca.h
/opt/homebrew/bin/gmkdir -p 'sys'
/opt/homebrew/bin/gmkdir -p 'sys'
sed -e 1h -e '1s,.*,/* DO NOT EDIT! GENERATED AUTOMATICALLY! */,' -e 1G \
	      -e 's|@''GUARD_PREFIX''@|GL|g' \
	      -e 's|@''INCLUDE_NEXT''@|include_next|g' \
	      -e 's|@''PRAGMA_SYSTEM_HEADER''@|#pragma GCC system_header|g' \
	      -e 's|@''PRAGMA_COLUMNS''@||g' \
	      -e 's|@''NEXT_TIME_H''@|<time.h>|g' \
	      -e 's/@''GNULIB_CTIME''@/0/g' \
	      -e 's/@''GNULIB_LOCALTIME''@/0/g' \
	      -e 's/@''GNULIB_MKTIME''@/0/g' \
	      -e 's/@''GNULIB_NANOSLEEP''@/0/g' \
	      -e 's/@''GNULIB_STRFTIME''@/0/g' \
	      -e 's/@''GNULIB_STRPTIME''@/0/g' \
	      -e 's/@''GNULIB_TIMEGM''@/0/g' \
	      -e 's/@''GNULIB_TIMESPEC_GET''@/0/g' \
	      -e 's/@''GNULIB_TIMESPEC_GETRES''@/0/g' \
	      -e 's/@''GNULIB_TIME_R''@/0/g' \
	      -e 's/@''GNULIB_TIME_RZ''@/0/g' \
	      -e 's/@''GNULIB_TZSET''@/0/g' \
	      -e 's/@''GNULIB_MDA_TZSET''@/1/g' \
	      -e 's|@''HAVE_DECL_LOCALTIME_R''@|1|g' \
	      -e 's|@''HAVE_NANOSLEEP''@|1|g' \
	      -e 's|@''HAVE_STRPTIME''@|1|g' \
	      -e 's|@''HAVE_TIMEGM''@|1|g' \
	      -e 's|@''HAVE_TIMESPEC_GET''@|1|g' \
	      -e 's|@''HAVE_TIMESPEC_GETRES''@|1|g' \
	      -e 's|@''HAVE_TIMEZONE_T''@|0|g' \
	      -e 's|@''REPLACE_CTIME''@|GNULIB_PORTCHECK|g' \
	      -e 's|@''REPLACE_GMTIME''@|0|g' \
	      -e 's|@''REPLACE_LOCALTIME''@|0|g' \
	      -e 's|@''REPLACE_LOCALTIME_R''@|GNULIB_PORTCHECK|g' \
	      -e 's|@''REPLACE_MKTIME''@|GNULIB_PORTCHECK|g' \
	      -e 's|@''REPLACE_NANOSLEEP''@|GNULIB_PORTCHECK|g' \
	      -e 's|@''REPLACE_STRFTIME''@|GNULIB_PORTCHECK|g' \
	      -e 's|@''REPLACE_TIMEGM''@|GNULIB_PORTCHECK|g' \
	      -e 's|@''REPLACE_TZSET''@|GNULIB_PORTCHECK|g' \
	      -e 's|@''PTHREAD_H_DEFINES_STRUCT_TIMESPEC''@|0|g' \
	      -e 's|@''SYS_TIME_H_DEFINES_STRUCT_TIMESPEC''@|0|g' \
	      -e 's|@''TIME_H_DEFINES_STRUCT_TIMESPEC''@|1|g' \
	      -e 's|@''UNISTD_H_DEFINES_STRUCT_TIMESPEC''@|0|g' \
	      -e 's|@''TIME_H_DEFINES_TIME_UTC''@|1|g' \
	      -e '/definitions of _GL_FUNCDECL_RPL/r ./c++defs.h' \
	      -e '/definition of _GL_ARG_NONNULL/r ./arg-nonnull.h' \
	      -e '/definition of _GL_WARN_ON_USE/r ./warn-on-use.h' \
	      ./time.in.h > time.h-t
sed -e 1h -e '1s,.*,/* DO NOT EDIT! GENERATED AUTOMATICALLY! */,' -e 1G \
	      -e 's|@''GUARD_PREFIX''@|GL|g' \
	      -e 's|@''HAVE_UNISTD_H''@|1|g' \
	      -e 's|@''INCLUDE_NEXT''@|include_next|g' \
	      -e 's|@''PRAGMA_SYSTEM_HEADER''@|#pragma GCC system_header|g' \
	      -e 's|@''PRAGMA_COLUMNS''@||g' \
	      -e 's|@''NEXT_UNISTD_H''@|<unistd.h>|g' \
	      -e 's|@''WINDOWS_64_BIT_OFF_T''@|0|g' \
	      -e 's/@''GNULIB_ACCESS''@/0/g' \
	      -e 's/@''GNULIB_CHDIR''@/0/g' \
	      -e 's/@''GNULIB_CHOWN''@/0/g' \
	      -e 's/@''GNULIB_CLOSE''@/1/g' \
	      -e 's/@''GNULIB_COPY_FILE_RANGE''@/0/g' \
	      -e 's/@''GNULIB_DUP''@/0/g' \
	      -e 's/@''GNULIB_DUP2''@/1/g' \
	      -e 's/@''GNULIB_DUP3''@/0/g' \
	      -e 's/@''GNULIB_ENVIRON''@/0/g' \
	      -e 's/@''GNULIB_EUIDACCESS''@/0/g' \
	      -e 's/@''GNULIB_EXECL''@/0/g' \
	      -e 's/@''GNULIB_EXECLE''@/0/g' \
	      -e 's/@''GNULIB_EXECLP''@/0/g' \
	      -e 's/@''GNULIB_EXECV''@/0/g' \
	      -e 's/@''GNULIB_EXECVE''@/0/g' \
	      -e 's/@''GNULIB_EXECVP''@/0/g' \
	      -e 's/@''GNULIB_EXECVPE''@/0/g' \
	      -e 's/@''GNULIB_FACCESSAT''@/0/g' \
	      -e 's/@''GNULIB_FCHDIR''@/0/g' \
	      -e 's/@''GNULIB_FCHOWNAT''@/0/g' \
	      -e 's/@''GNULIB_FDATASYNC''@/0/g' \
	      -e 's/@''GNULIB_FSYNC''@/0/g' \
	      -e 's/@''GNULIB_FTRUNCATE''@/0/g' \
	      -e 's/@''GNULIB_GETCWD''@/0/g' \
	      -e 's/@''GNULIB_GETDOMAINNAME''@/0/g' \
	      -e 's/@''GNULIB_GETDTABLESIZE''@/1/g' \
	      -e 's/@''GNULIB_GETENTROPY''@/0/g' \
	      -e 's/@''GNULIB_GETGROUPS''@/0/g' \
	      -e 's/@''GNULIB_GETHOSTNAME''@/0/g' \
	      -e 's/@''GNULIB_GETLOGIN''@/0/g' \
	      -e 's/@''GNULIB_GETLOGIN_R''@/0/g' \
	      -e 's/@''GNULIB_GETOPT_POSIX''@/1/g' \
	      -e 's/@''GNULIB_GETPAGESIZE''@/0/g' \
	      -e 's/@''GNULIB_GETPASS''@/0/g' \
	      -e 's/@''GNULIB_GETPASS_GNU''@/0/g' \
	      -e 's/@''GNULIB_GETUSERSHELL''@/0/g' \
	      -e 's/@''GNULIB_GROUP_MEMBER''@/0/g' \
	      -e 's/@''GNULIB_ISATTY''@/0/g' \
	      -e 's/@''GNULIB_LCHOWN''@/0/g' \
	      -e 's/@''GNULIB_LINK''@/0/g' \
	      -e 's/@''GNULIB_LINKAT''@/0/g' \
	      -e 's/@''GNULIB_LSEEK''@/0/g' \
	      -e 's/@''GNULIB_PIPE''@/0/g' \
	      -e 's/@''GNULIB_PIPE2''@/0/g' \
	      -e 's/@''GNULIB_PREAD''@/0/g' \
	      -e 's/@''GNULIB_PWRITE''@/0/g' \
	      -e 's/@''GNULIB_READ''@/0/g' \
	      -e 's/@''GNULIB_READLINK''@/0/g' \
	      -e 's/@''GNULIB_READLINKAT''@/0/g' \
	      -e 's/@''GNULIB_RMDIR''@/0/g' \
	      -e 's/@''GNULIB_SETHOSTNAME''@/0/g' \
	      -e 's/@''GNULIB_SLEEP''@/0/g' \
	      -e 's/@''GNULIB_SYMLINK''@/0/g' \
	      -e 's/@''GNULIB_SYMLINKAT''@/0/g' \
	      -e 's/@''GNULIB_TRUNCATE''@/0/g' \
	      -e 's/@''GNULIB_TTYNAME_R''@/0/g' \
	      -e 's/@''GNULIB_UNISTD_H_GETOPT''@/01/g' \
	      -e 's/@''GNULIB_UNISTD_H_NONBLOCKING''@/0/g' \
	      -e 's/@''GNULIB_UNISTD_H_SIGPIPE''@/0/g' \
	      -e 's/@''GNULIB_UNLINK''@/0/g' \
	      -e 's/@''GNULIB_UNLINKAT''@/0/g' \
	      -e 's/@''GNULIB_USLEEP''@/0/g' \
	      -e 's/@''GNULIB_WRITE''@/0/g' \
	      -e 's/@''GNULIB_MDA_ACCESS''@/1/g' \
	      -e 's/@''GNULIB_MDA_CHDIR''@/1/g' \
	      -e 's/@''GNULIB_MDA_CLOSE''@/1/g' \
	      -e 's/@''GNULIB_MDA_DUP''@/1/g' \
	      -e 's/@''GNULIB_MDA_DUP2''@/1/g' \
	      -e 's/@''GNULIB_MDA_EXECL''@/1/g' \
	      -e 's/@''GNULIB_MDA_EXECLE''@/1/g' \
	      -e 's/@''GNULIB_MDA_EXECLP''@/1/g' \
	      -e 's/@''GNULIB_MDA_EXECV''@/1/g' \
	      -e 's/@''GNULIB_MDA_EXECVE''@/1/g' \
	      -e 's/@''GNULIB_MDA_EXECVP''@/1/g' \
	      -e 's/@''GNULIB_MDA_EXECVPE''@/1/g' \
	      -e 's/@''GNULIB_MDA_GETCWD''@/1/g' \
	      -e 's/@''GNULIB_MDA_GETPID''@/1/g' \
	      -e 's/@''GNULIB_MDA_ISATTY''@/1/g' \
	      -e 's/@''GNULIB_MDA_LSEEK''@/1/g' \
	      -e 's/@''GNULIB_MDA_READ''@/1/g' \
	      -e 's/@''GNULIB_MDA_RMDIR''@/1/g' \
	      -e 's/@''GNULIB_MDA_SWAB''@/1/g' \
	      -e 's/@''GNULIB_MDA_UNLINK''@/1/g' \
	      -e 's/@''GNULIB_MDA_WRITE''@/1/g' \
	      < ./unistd.in.h | \
	  sed -e 's|@''HAVE_CHOWN''@|1|g' \
	      -e 's|@''HAVE_COPY_FILE_RANGE''@|1|g' \
	      -e 's|@''HAVE_DUP3''@|1|g' \
	      -e 's|@''HAVE_EUIDACCESS''@|1|g' \
	      -e 's|@''HAVE_EXECVPE''@|1|g' \
	      -e 's|@''HAVE_FACCESSAT''@|1|g' \
	      -e 's|@''HAVE_FCHDIR''@|1|g' \
	      -e 's|@''HAVE_FCHOWNAT''@|1|g' \
	      -e 's|@''HAVE_FDATASYNC''@|1|g' \
	      -e 's|@''HAVE_FSYNC''@|1|g' \
	      -e 's|@''HAVE_FTRUNCATE''@|1|g' \
	      -e 's|@''HAVE_GETDTABLESIZE''@|1|g' \
	      -e 's|@''HAVE_GETENTROPY''@|1|g' \
	      -e 's|@''HAVE_GETGROUPS''@|1|g' \
	      -e 's|@''HAVE_GETHOSTNAME''@|1|g' \
	      -e 's|@''HAVE_GETPAGESIZE''@|1|g' \
	      -e 's|@''HAVE_GETPASS''@|1|g' \
	      -e 's|@''HAVE_GROUP_MEMBER''@|1|g' \
	      -e 's|@''HAVE_LCHOWN''@|1|g' \
	      -e 's|@''HAVE_LINK''@|1|g' \
	      -e 's|@''HAVE_LINKAT''@|1|g' \
	      -e 's|@''HAVE_PIPE''@|1|g' \
	      -e 's|@''HAVE_PIPE2''@|1|g' \
	      -e 's|@''HAVE_PREAD''@|1|g' \
	      -e 's|@''HAVE_PWRITE''@|1|g' \
	      -e 's|@''HAVE_READLINK''@|1|g' \
	      -e 's|@''HAVE_READLINKAT''@|1|g' \
	      -e 's|@''HAVE_SETHOSTNAME''@|1|g' \
	      -e 's|@''HAVE_SLEEP''@|1|g' \
	      -e 's|@''HAVE_SYMLINK''@|1|g' \
	      -e 's|@''HAVE_SYMLINKAT''@|1|g' \
	      -e 's|@''HAVE_UNLINKAT''@|1|g' \
	      -e 's|@''HAVE_USLEEP''@|1|g' \
	      -e 's|@''HAVE_DECL_ENVIRON''@|1|g' \
	      -e 's|@''HAVE_DECL_EXECVPE''@|0|g' \
	      -e 's|@''HAVE_DECL_FCHDIR''@|1|g' \
	      -e 's|@''HAVE_DECL_FDATASYNC''@|1|g' \
	      -e 's|@''HAVE_DECL_GETDOMAINNAME''@|1|g' \
	      -e 's|@''HAVE_DECL_GETLOGIN''@|1|g' \
	      -e 's|@''HAVE_DECL_GETLOGIN_R''@|1|g' \
	      -e 's|@''HAVE_DECL_GETPAGESIZE''@|1|g' \
	      -e 's|@''HAVE_DECL_GETUSERSHELL''@|1|g' \
	      -e 's|@''HAVE_DECL_SETHOSTNAME''@|1|g' \
	      -e 's|@''HAVE_DECL_TRUNCATE''@|1|g' \
	      -e 's|@''HAVE_DECL_TTYNAME_R''@|1|g' \
	      -e 's|@''HAVE_OS_H''@|0|g' \
	      -e 's|@''HAVE_SYS_PARAM_H''@|0|g' \
	  | \
	  sed -e 's|@''REPLACE_ACCESS''@|0|g' \
	      -e 's|@''REPLACE_CHOWN''@|0|g' \
	      -e 's|@''REPLACE_CLOSE''@|0|g' \
	      -e 's|@''REPLACE_COPY_FILE_RANGE''@|0|g' \
	      -e 's|@''REPLACE_DUP''@|0|g' \
	      -e 's|@''REPLACE_DUP2''@|0|g' \
	      -e 's|@''REPLACE_EXECL''@|0|g' \
	      -e 's|@''REPLACE_EXECLE''@|0|g' \
	      -e 's|@''REPLACE_EXECLP''@|0|g' \
	      -e 's|@''REPLACE_EXECV''@|0|g' \
	      -e 's|@''REPLACE_EXECVE''@|0|g' \
	      -e 's|@''REPLACE_EXECVP''@|0|g' \
	      -e 's|@''REPLACE_EXECVPE''@|0|g' \
	      -e 's|@''REPLACE_FACCESSAT''@|0|g' \
	      -e 's|@''REPLACE_FCHOWNAT''@|0|g' \
	      -e 's|@''REPLACE_FTRUNCATE''@|0|g' \
	      -e 's|@''REPLACE_GETCWD''@|0|g' \
	      -e 's|@''REPLACE_GETDOMAINNAME''@|0|g' \
	      -e 's|@''REPLACE_GETDTABLESIZE''@|0|g' \
	      -e 's|@''REPLACE_GETLOGIN_R''@|0|g' \
	      -e 's|@''REPLACE_GETGROUPS''@|0|g' \
	      -e 's|@''REPLACE_GETPAGESIZE''@|0|g' \
	      -e 's|@''REPLACE_GETPASS''@|0|g' \
	      -e 's|@''REPLACE_GETPASS_FOR_GETPASS_GNU''@|0|g' \
	      -e 's|@''REPLACE_ISATTY''@|0|g' \
	      -e 's|@''REPLACE_LCHOWN''@|0|g' \
	      -e 's|@''REPLACE_LINK''@|0|g' \
	      -e 's|@''REPLACE_LINKAT''@|0|g' \
	      -e 's|@''REPLACE_LSEEK''@|0|g' \
	      -e 's|@''REPLACE_PREAD''@|0|g' \
	      -e 's|@''REPLACE_PWRITE''@|0|g' \
	      -e 's|@''REPLACE_READ''@|0|g' \
	      -e 's|@''REPLACE_READLINK''@|0|g' \
	      -e 's|@''REPLACE_READLINKAT''@|0|g' \
	      -e 's|@''REPLACE_RMDIR''@|0|g' \
	      -e 's|@''REPLACE_SLEEP''@|0|g' \
	      -e 's|@''REPLACE_SYMLINK''@|0|g' \
	      -e 's|@''REPLACE_SYMLINKAT''@|0|g' \
	      -e 's|@''REPLACE_TRUNCATE''@|0|g' \
	      -e 's|@''REPLACE_TTYNAME_R''@|0|g' \
	      -e 's|@''REPLACE_UNLINK''@|0|g' \
	      -e 's|@''REPLACE_UNLINKAT''@|0|g' \
	      -e 's|@''REPLACE_USLEEP''@|0|g' \
	      -e 's|@''REPLACE_WRITE''@|0|g' \
	      -e 's|@''UNISTD_H_HAVE_SYS_RANDOM_H''@|0|g' \
	      -e 's|@''UNISTD_H_HAVE_WINSOCK2_H''@|0|g' \
	      -e 's|@''UNISTD_H_HAVE_WINSOCK2_H_AND_USE_SOCKETS''@|0|g' \
	      -e '/definitions of _GL_FUNCDECL_RPL/r ./c++defs.h' \
	      -e '/definition of _GL_ARG_NONNULL/r ./arg-nonnull.h' \
	      -e '/definition of _GL_WARN_ON_USE/r ./warn-on-use.h' \
	      > unistd.h-t
sed -e 1h -e '1s,.*,/* DO NOT EDIT! GENERATED AUTOMATICALLY! */,' -e 1G \
	      -e 's|@''GUARD_PREFIX''@|GL|g' \
	      -e 's|@''INCLUDE_NEXT''@|include_next|g' \
	      -e 's|@''PRAGMA_SYSTEM_HEADER''@|#pragma GCC system_header|g' \
	      -e 's|@''PRAGMA_COLUMNS''@||g' \
	      -e 's|@''NEXT_SYS_TYPES_H''@|<sys/types.h>|g' \
	      -e 's|@''WINDOWS_64_BIT_OFF_T''@|0|g' \
	      -e 's|@''WINDOWS_STAT_INODES''@|0|g' \
	      ./sys_types.in.h > sys/types.h-t
sed -e 1h -e '1s,.*,/* DO NOT EDIT! GENERATED AUTOMATICALLY! */,' -e 1G \
	      -e 's|@''GUARD_PREFIX''@|GL|g' \
	      -e 's|@''INCLUDE_NEXT''@|include_next|g' \
	      -e 's|@''PRAGMA_SYSTEM_HEADER''@|#pragma GCC system_header|g' \
	      -e 's|@''PRAGMA_COLUMNS''@||g' \
	      -e 's|@''NEXT_SYS_STAT_H''@|<sys/stat.h>|g' \
	      -e 's|@''WINDOWS_64_BIT_ST_SIZE''@|0|g' \
	      -e 's|@''WINDOWS_STAT_TIMESPEC''@|0|g' \
	      -e 's/@''GNULIB_CHMOD''@/0/g' \
	      -e 's/@''GNULIB_FCHMODAT''@/0/g' \
	      -e 's/@''GNULIB_FSTAT''@/1/g' \
	      -e 's/@''GNULIB_FSTATAT''@/0/g' \
	      -e 's/@''GNULIB_FUTIMENS''@/0/g' \
	      -e 's/@''GNULIB_GETUMASK''@/0/g' \
	      -e 's/@''GNULIB_LCHMOD''@/0/g' \
	      -e 's/@''GNULIB_LSTAT''@/0/g' \
	      -e 's/@''GNULIB_MKDIR''@/0/g' \
	      -e 's/@''GNULIB_MKDIRAT''@/0/g' \
	      -e 's/@''GNULIB_MKFIFO''@/0/g' \
	      -e 's/@''GNULIB_MKFIFOAT''@/0/g' \
	      -e 's/@''GNULIB_MKNOD''@/0/g' \
	      -e 's/@''GNULIB_MKNODAT''@/0/g' \
	      -e 's/@''GNULIB_STAT''@/1/g' \
	      -e 's/@''GNULIB_UTIMENSAT''@/0/g' \
	      -e 's/@''GNULIB_OVERRIDES_STRUCT_STAT''@/0/g' \
	      -e 's/@''GNULIB_MDA_CHMOD''@/1/g' \
	      -e 's/@''GNULIB_MDA_MKDIR''@/1/g' \
	      -e 's/@''GNULIB_MDA_UMASK''@/1/g' \
	      -e 's|@''HAVE_FCHMODAT''@|1|g' \
	      -e 's|@''HAVE_FSTATAT''@|1|g' \
	      -e 's|@''HAVE_FUTIMENS''@|1|g' \
	      -e 's|@''HAVE_GETUMASK''@|1|g' \
	      -e 's|@''HAVE_LCHMOD''@|1|g' \
	      -e 's|@''HAVE_LSTAT''@|1|g' \
	      -e 's|@''HAVE_MKDIRAT''@|1|g' \
	      -e 's|@''HAVE_MKFIFO''@|1|g' \
	      -e 's|@''HAVE_MKFIFOAT''@|1|g' \
	      -e 's|@''HAVE_MKNOD''@|1|g' \
	      -e 's|@''HAVE_MKNODAT''@|1|g' \
	      -e 's|@''HAVE_UTIMENSAT''@|1|g' \
	      -e 's|@''REPLACE_CHMOD''@|0|g' \
	      -e 's|@''REPLACE_FCHMODAT''@|0|g' \
	      -e 's|@''REPLACE_FSTAT''@|0|g' \
	      -e 's|@''REPLACE_FSTATAT''@|0|g' \
	      -e 's|@''REPLACE_FUTIMENS''@|0|g' \
	      -e 's|@''REPLACE_LSTAT''@|0|g' \
	      -e 's|@''REPLACE_MKDIR''@|0|g' \
	      -e 's|@''REPLACE_MKFIFO''@|0|g' \
	      -e 's|@''REPLACE_MKFIFOAT''@|0|g' \
	      -e 's|@''REPLACE_MKNOD''@|0|g' \
	      -e 's|@''REPLACE_MKNODAT''@|0|g' \
	      -e 's|@''REPLACE_STAT''@|1|g' \
	      -e 's|@''REPLACE_UTIMENSAT''@|0|g' \
	      -e '/definitions of _GL_FUNCDECL_RPL/r ./c++defs.h' \
	      -e '/definition of _GL_ARG_NONNULL/r ./arg-nonnull.h' \
	      -e '/definition of _GL_WARN_ON_USE/r ./warn-on-use.h' \
	      ./sys_stat.in.h > sys/stat.h-t
mv sys/types.h-t sys/types.h
mv sys/stat.h-t sys/stat.h
mv time.h-t time.h
mv unistd.h-t unistd.h
/Applications/Xcode.app/Contents/Developer/usr/bin/make  all-recursive
/bin/sh ../libtool  --tag=CC   --mode=compile /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g -DHAVE_CONFIG_H -I. -I..   -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -DIDN2_STATIC -fvisibility=hidden -Wno-cast-qual -Wno-conversion -Wno-float-equal -Wno-sign-compare -Wno-undef -Wno-unused-function -Wno-unused-parameter -Wno-float-conversion -Wimplicit-fallthrough -Wno-pedantic -Wno-sign-conversion -Wno-type-limits -fPIC -g -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/x64-osx/include -MT libgnu_la-basename-lgpl.lo -MD -MP -MF .deps/libgnu_la-basename-lgpl.Tpo -c -o libgnu_la-basename-lgpl.lo `test -f 'basename-lgpl.c' || echo './'`basename-lgpl.c
/bin/sh ../libtool  --tag=CC   --mode=compile /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g -DHAVE_CONFIG_H -I. -I..   -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -DIDN2_STATIC -fvisibility=hidden -Wno-cast-qual -Wno-conversion -Wno-float-equal -Wno-sign-compare -Wno-undef -Wno-unused-function -Wno-unused-parameter -Wno-float-conversion -Wimplicit-fallthrough -Wno-pedantic -Wno-sign-conversion -Wno-type-limits -fPIC -g -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/x64-osx/include -MT libgnu_la-cloexec.lo -MD -MP -MF .deps/libgnu_la-cloexec.Tpo -c -o libgnu_la-cloexec.lo `test -f 'cloexec.c' || echo './'`cloexec.c
/bin/sh ../libtool  --tag=CC   --mode=compile /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g -DHAVE_CONFIG_H -I. -I..   -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -DIDN2_STATIC -fvisibility=hidden -Wno-cast-qual -Wno-conversion -Wno-float-equal -Wno-sign-compare -Wno-undef -Wno-unused-function -Wno-unused-parameter -Wno-float-conversion -Wimplicit-fallthrough -Wno-pedantic -Wno-sign-conversion -Wno-type-limits -fPIC -g -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/x64-osx/include -MT libgnu_la-error.lo -MD -MP -MF .deps/libgnu_la-error.Tpo -c -o libgnu_la-error.lo `test -f 'error.c' || echo './'`error.c
/bin/sh ../libtool  --tag=CC   --mode=compile /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g -DHAVE_CONFIG_H -I. -I..   -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -DIDN2_STATIC -fvisibility=hidden -Wno-cast-qual -Wno-conversion -Wno-float-equal -Wno-sign-compare -Wno-undef -Wno-unused-function -Wno-unused-parameter -Wno-float-conversion -Wimplicit-fallthrough -Wno-pedantic -Wno-sign-conversion -Wno-type-limits -fPIC -g -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/x64-osx/include -MT libgnu_la-fd-hook.lo -MD -MP -MF .deps/libgnu_la-fd-hook.Tpo -c -o libgnu_la-fd-hook.lo `test -f 'fd-hook.c' || echo './'`fd-hook.c
/bin/sh ../libtool  --tag=CC   --mode=compile /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g -DHAVE_CONFIG_H -I. -I..   -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -DIDN2_STATIC -fvisibility=hidden -Wno-cast-qual -Wno-conversion -Wno-float-equal -Wno-sign-compare -Wno-undef -Wno-unused-function -Wno-unused-parameter -Wno-float-conversion -Wimplicit-fallthrough -Wno-pedantic -Wno-sign-conversion -Wno-type-limits -fPIC -g -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/x64-osx/include -MT libgnu_la-getopt.lo -MD -MP -MF .deps/libgnu_la-getopt.Tpo -c -o libgnu_la-getopt.lo `test -f 'getopt.c' || echo './'`getopt.c
/bin/sh ../libtool  --tag=CC   --mode=compile /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g -DHAVE_CONFIG_H -I. -I..   -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -DIDN2_STATIC -fvisibility=hidden -Wno-cast-qual -Wno-conversion -Wno-float-equal -Wno-sign-compare -Wno-undef -Wno-unused-function -Wno-unused-parameter -Wno-float-conversion -Wimplicit-fallthrough -Wno-pedantic -Wno-sign-conversion -Wno-type-limits -fPIC -g -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/x64-osx/include -MT libgnu_la-getopt1.lo -MD -MP -MF .deps/libgnu_la-getopt1.Tpo -c -o libgnu_la-getopt1.lo `test -f 'getopt1.c' || echo './'`getopt1.c
/bin/sh ../libtool  --tag=CC   --mode=compile /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g -DHAVE_CONFIG_H -I. -I..   -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -DIDN2_STATIC -fvisibility=hidden -Wno-cast-qual -Wno-conversion -Wno-float-equal -Wno-sign-compare -Wno-undef -Wno-unused-function -Wno-unused-parameter -Wno-float-conversion -Wimplicit-fallthrough -Wno-pedantic -Wno-sign-conversion -Wno-type-limits -fPIC -g -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/x64-osx/include -MT libgnu_la-getprogname.lo -MD -MP -MF .deps/libgnu_la-getprogname.Tpo -c -o libgnu_la-getprogname.lo `test -f 'getprogname.c' || echo './'`getprogname.c
/bin/sh ../libtool  --tag=CC   --mode=compile /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g -DHAVE_CONFIG_H -I. -I..   -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -DIDN2_STATIC -fvisibility=hidden -Wno-cast-qual -Wno-conversion -Wno-float-equal -Wno-sign-compare -Wno-undef -Wno-unused-function -Wno-unused-parameter -Wno-float-conversion -Wimplicit-fallthrough -Wno-pedantic -Wno-sign-conversion -Wno-type-limits -fPIC -g -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/x64-osx/include -MT libgnu_la-malloca.lo -MD -MP -MF .deps/libgnu_la-malloca.Tpo -c -o libgnu_la-malloca.lo `test -f 'malloca.c' || echo './'`malloca.c
/bin/sh ../libtool  --tag=CC   --mode=compile /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g -DHAVE_CONFIG_H -I. -I..   -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -DIDN2_STATIC -fvisibility=hidden -Wno-cast-qual -Wno-conversion -Wno-float-equal -Wno-sign-compare -Wno-undef -Wno-unused-function -Wno-unused-parameter -Wno-float-conversion -Wimplicit-fallthrough -Wno-pedantic -Wno-sign-conversion -Wno-type-limits -fPIC -g -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/x64-osx/include -MT libgnu_la-open.lo -MD -MP -MF .deps/libgnu_la-open.Tpo -c -o libgnu_la-open.lo `test -f 'open.c' || echo './'`open.c
/bin/sh ../libtool  --tag=CC   --mode=compile /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g -DHAVE_CONFIG_H -I. -I..   -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -DIDN2_STATIC -fvisibility=hidden -Wno-cast-qual -Wno-conversion -Wno-float-equal -Wno-sign-compare -Wno-undef -Wno-unused-function -Wno-unused-parameter -Wno-float-conversion -Wimplicit-fallthrough -Wno-pedantic -Wno-sign-conversion -Wno-type-limits -fPIC -g -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/x64-osx/include -MT libgnu_la-progname.lo -MD -MP -MF .deps/libgnu_la-progname.Tpo -c -o libgnu_la-progname.lo `test -f 'progname.c' || echo './'`progname.c
/bin/sh ../libtool  --tag=CC   --mode=compile /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g -DHAVE_CONFIG_H -I. -I..   -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -DIDN2_STATIC -fvisibility=hidden -Wno-cast-qual -Wno-conversion -Wno-float-equal -Wno-sign-compare -Wno-undef -Wno-unused-function -Wno-unused-parameter -Wno-float-conversion -Wimplicit-fallthrough -Wno-pedantic -Wno-sign-conversion -Wno-type-limits -fPIC -g -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/x64-osx/include -MT libgnu_la-rawmemchr.lo -MD -MP -MF .deps/libgnu_la-rawmemchr.Tpo -c -o libgnu_la-rawmemchr.lo `test -f 'rawmemchr.c' || echo './'`rawmemchr.c
libtool: compile:  /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g -DHAVE_CONFIG_H -I. -I.. -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -DIDN2_STATIC -fvisibility=hidden -Wno-cast-qual -Wno-conversion -Wno-float-equal -Wno-sign-compare -Wno-undef -Wno-unused-function -Wno-unused-parameter -Wno-float-conversion -Wimplicit-fallthrough -Wno-pedantic -Wno-sign-conversion -Wno-type-limits -fPIC -g -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/x64-osx/include -MT libgnu_la-rawmemchr.lo -MD -MP -MF .deps/libgnu_la-rawmemchr.Tpo -c rawmemchr.c -o libgnu_la-rawmemchr.o
libtool: compile:  /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g -DHAVE_CONFIG_H -I. -I.. -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -DIDN2_STATIC -fvisibility=hidden -Wno-cast-qual -Wno-conversion -Wno-float-equal -Wno-sign-compare -Wno-undef -Wno-unused-function -Wno-unused-parameter -Wno-float-conversion -Wimplicit-fallthrough -Wno-pedantic -Wno-sign-conversion -Wno-type-limits -fPIC -g -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/x64-osx/include -MT libgnu_la-cloexec.lo -MD -MP -MF .deps/libgnu_la-cloexec.Tpo -c cloexec.c -o libgnu_la-cloexec.o
libtool: compile:  /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g -DHAVE_CONFIG_H -I. -I.. -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -DIDN2_STATIC -fvisibility=hidden -Wno-cast-qual -Wno-conversion -Wno-float-equal -Wno-sign-compare -Wno-undef -Wno-unused-function -Wno-unused-parameter -Wno-float-conversion -Wimplicit-fallthrough -Wno-pedantic -Wno-sign-conversion -Wno-type-limits -fPIC -g -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/x64-osx/include -MT libgnu_la-fd-hook.lo -MD -MP -MF .deps/libgnu_la-fd-hook.Tpo -c fd-hook.c -o libgnu_la-fd-hook.o
libtool: compile:  /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g -DHAVE_CONFIG_H -I. -I.. -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -DIDN2_STATIC -fvisibility=hidden -Wno-cast-qual -Wno-conversion -Wno-float-equal -Wno-sign-compare -Wno-undef -Wno-unused-function -Wno-unused-parameter -Wno-float-conversion -Wimplicit-fallthrough -Wno-pedantic -Wno-sign-conversion -Wno-type-limits -fPIC -g -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/x64-osx/include -MT libgnu_la-getopt.lo -MD -MP -MF .deps/libgnu_la-getopt.Tpo -c getopt.c -o libgnu_la-getopt.o
libtool: compile:  /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g -DHAVE_CONFIG_H -I. -I.. -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -DIDN2_STATIC -fvisibility=hidden -Wno-cast-qual -Wno-conversion -Wno-float-equal -Wno-sign-compare -Wno-undef -Wno-unused-function -Wno-unused-parameter -Wno-float-conversion -Wimplicit-fallthrough -Wno-pedantic -Wno-sign-conversion -Wno-type-limits -fPIC -g -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/x64-osx/include -MT libgnu_la-error.lo -MD -MP -MF .deps/libgnu_la-error.Tpo -c error.c -o libgnu_la-error.o
libtool: compile:  /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g -DHAVE_CONFIG_H -I. -I.. -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -DIDN2_STATIC -fvisibility=hidden -Wno-cast-qual -Wno-conversion -Wno-float-equal -Wno-sign-compare -Wno-undef -Wno-unused-function -Wno-unused-parameter -Wno-float-conversion -Wimplicit-fallthrough -Wno-pedantic -Wno-sign-conversion -Wno-type-limits -fPIC -g -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/x64-osx/include -MT libgnu_la-progname.lo -MD -MP -MF .deps/libgnu_la-progname.Tpo -c progname.c -o libgnu_la-progname.o
libtool: compile:  /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g -DHAVE_CONFIG_H -I. -I.. -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -DIDN2_STATIC -fvisibility=hidden -Wno-cast-qual -Wno-conversion -Wno-float-equal -Wno-sign-compare -Wno-undef -Wno-unused-function -Wno-unused-parameter -Wno-float-conversion -Wimplicit-fallthrough -Wno-pedantic -Wno-sign-conversion -Wno-type-limits -fPIC -g -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/x64-osx/include -MT libgnu_la-open.lo -MD -MP -MF .deps/libgnu_la-open.Tpo -c open.c -o libgnu_la-open.o
libtool: compile:  /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g -DHAVE_CONFIG_H -I. -I.. -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -DIDN2_STATIC -fvisibility=hidden -Wno-cast-qual -Wno-conversion -Wno-float-equal -Wno-sign-compare -Wno-undef -Wno-unused-function -Wno-unused-parameter -Wno-float-conversion -Wimplicit-fallthrough -Wno-pedantic -Wno-sign-conversion -Wno-type-limits -fPIC -g -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/x64-osx/include -MT libgnu_la-getprogname.lo -MD -MP -MF .deps/libgnu_la-getprogname.Tpo -c getprogname.c -o libgnu_la-getprogname.o
libtool: compile:  /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g -DHAVE_CONFIG_H -I. -I.. -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -DIDN2_STATIC -fvisibility=hidden -Wno-cast-qual -Wno-conversion -Wno-float-equal -Wno-sign-compare -Wno-undef -Wno-unused-function -Wno-unused-parameter -Wno-float-conversion -Wimplicit-fallthrough -Wno-pedantic -Wno-sign-conversion -Wno-type-limits -fPIC -g -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/x64-osx/include -MT libgnu_la-getopt1.lo -MD -MP -MF .deps/libgnu_la-getopt1.Tpo -c getopt1.c -o libgnu_la-getopt1.o
libtool: compile:  /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g -DHAVE_CONFIG_H -I. -I.. -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -DIDN2_STATIC -fvisibility=hidden -Wno-cast-qual -Wno-conversion -Wno-float-equal -Wno-sign-compare -Wno-undef -Wno-unused-function -Wno-unused-parameter -Wno-float-conversion -Wimplicit-fallthrough -Wno-pedantic -Wno-sign-conversion -Wno-type-limits -fPIC -g -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/x64-osx/include -MT libgnu_la-malloca.lo -MD -MP -MF .deps/libgnu_la-malloca.Tpo -c malloca.c -o libgnu_la-malloca.o
libtool: compile:  /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g -DHAVE_CONFIG_H -I. -I.. -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -DIDN2_STATIC -fvisibility=hidden -Wno-cast-qual -Wno-conversion -Wno-float-equal -Wno-sign-compare -Wno-undef -Wno-unused-function -Wno-unused-parameter -Wno-float-conversion -Wimplicit-fallthrough -Wno-pedantic -Wno-sign-conversion -Wno-type-limits -fPIC -g -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/x64-osx/include -MT libgnu_la-basename-lgpl.lo -MD -MP -MF .deps/libgnu_la-basename-lgpl.Tpo -c basename-lgpl.c -o libgnu_la-basename-lgpl.o
mv -f .deps/libgnu_la-basename-lgpl.Tpo .deps/libgnu_la-basename-lgpl.Plo
```
</details>
<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libidn2/build-x64-osx-dbg-err.log</summary>

```
In file included from fd-hook.c:23:
In file included from ./stdlib.h:36:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/stdlib.h:66:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/sys/wait.h:110:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/sys/resource.h:72:
./stdint.h:89:5: error: expected value in expression
#if 
    ^
1 error generated.
In file included from rawmemchr.c:26:
./stdint.h:89:5: error: expected value in expression
#if 
    ^
In file included from cloexec.c:25:
In file included from ./fcntl.h:79:
In file included from ./unistd.h:93:
In file included from ./stdlib.h:28:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/stdlib.h:66:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/sys/wait.h:110:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/sys/resource.h:72:
./stdint.h:89:5: error: expected value in expression
#if 
    ^
1 error generated.
1 error generated.
In file included from progname.c:27:
In file included from ./stdlib.h:36:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/stdlib.h:66:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/sys/wait.h:110:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/sys/resource.h:72:
./stdint.h:89:5: error: expected value in expression
#if 
    ^
1 error generated.
In file included from getopt.c:24:
In file included from ./getopt.h:34:
In file included from ./getopt.h:34:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/getopt.h:44:
In file included from ./unistd.h:93:
In file included from ./stdlib.h:28:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/stdlib.h:66:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/sys/wait.h:110:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/sys/resource.h:72:
./stdint.h:89:5: error: expected value in expression
#if 
    ^
In file included from error.c:28:
In file included from ./stdlib.h:36:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/stdlib.h:66:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/sys/wait.h:110:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/sys/resource.h:72:
./stdint.h:89:5: error: expected value in expression
#if 
    ^
In file included from getopt1.c:24:
In file included from ./getopt.h:34:
In file included from ./getopt.h:34:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/getopt.h:44:
In file included from ./unistd.h:93:
In file included from ./stdlib.h:28:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/stdlib.h:66:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/sys/wait.h:110:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/sys/resource.h:72:
./stdint.h:89:5: error: expected value in expression
#if 
    ^
In file included from getprogname.c:20:
In file included from ./getprogname.h:20:
In file included from ./stdlib.h:36:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/stdlib.h:66In file included from malloca.c:22:
In file included from ./malloca.h:23:
In file included from ./stdlib.h:36:
In file included from :
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/sys/wait.h:110:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/sys/resource.h:72:
./stdint.h:89:5: error: expected value in expression
#if 
    ^
1 error generated.
1 error generated.
1 error generated.
In file included from open.c:43:
In file included from ./fcntl.h:67:
In file included from ./fcntl.h:79:
In file included from ./unistd.h:93:
In file included from ./stdlib.h:28:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/stdlib.h:66:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/sys/wait.h:110:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/sys/resource.h:72:
./stdint.h:89:5: error: expected value in expression
#if 
    ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/stdlib.h:66:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/sys/wait.h:110:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/sys/resource.h:72:
./stdint.h:89:5: error: expected value in expression
#if 
    ^
make[4]: *** [libgnu_la-rawmemchr.lo] Error 1
make[4]: *** Waiting for unfinished jobs....
make[4]: *** [libgnu_la-fd-hook.lo] Error 1
make[4]: *** [libgnu_la-cloexec.lo] Error 1
make[4]: *** [libgnu_la-progname.lo] Error 1
1 error generated.
1 error generated.
1 error generated.
make[4]: *** [libgnu_la-getopt.lo] Error 1
make[4]: *** [libgnu_la-open.lo] Error 1
make[4]: *** [libgnu_la-error.lo] Error 1
make[4]: *** [libgnu_la-malloca.lo] Error 1
make[4]: *** [libgnu_la-getprogname.lo] Error 1
make[4]: *** [libgnu_la-getopt1.lo] Error 1
make[3]: *** [all-recursive] Error 1
make[2]: *** [all] Error 2
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
```
</details>

**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
  "name": "lichtsteuerung",
  "version-string": "2.0 $.0",
  "dependencies": [
    {
      "name": "libidn2",
      "default-features": false,
      "features": [
        "nls"
      ]
    }
  ]
}

```
</details>

This issue could be reproduced with vcpkg 2023-04-07-bedcba5172f5e4b91caac660ab7afe92c27a9895:
````
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /usr/bin/make V=1 -j 13 -f Makefile all
    Working Directory: /Users/vcpkg/Jon/vcpkg/buildtrees/libidn2/x64-osx-dbg/
    See logs for more information:
      /Users/vcpkg/Jon/vcpkg/buildtrees/libidn2/build-x64-osx-dbg-out.log
      /Users/vcpkg/Jon/vcpkg/buildtrees/libidn2/build-x64-osx-dbg-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_make.cmake:151 (vcpkg_execute_build_process)
  scripts/cmake/vcpkg_install_make.cmake:2 (vcpkg_build_make)
  ports/libidn2/portfile.cmake:60 (vcpkg_install_make)
  scripts/ports.cmake:147 (include)


error: building libidn2:x64-osx failed with: BUILD_FAILED
Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
Then check for known issues at:
    https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+libidn2
You can submit a new issue at:
    https://github.com/microsoft/vcpkg/issues/new?title=[libidn2]+Build+error&body=Copy+issue+body+from+%2FUsers%2Fvcpkg%2FJon%2Fvcpkg%2Fvcpkg_installed%2Fvcpkg%2Fissue_body.md

vcpkg@VCPKGMM-12 vcpkg % ./vcpkg --version
vcpkg package management program version 2023-04-07-bedcba5172f5e4b91caac660ab7afe92c27a9895

See LICENSE.txt for license information.
vcpkg@VCPKGMM-12 vcpkg % vim /Users/vcpkg/Jon/vcpkg/buildtrees/libidn2/build-x64-osx-dbg-err.log

````

--- a/ports/libidn2/portfile.cmake
+++ b/ports/libidn2/portfile.cmake
@@ -29,9 +29,9 @@ vcpkg_list(SET options)
 if("nls" IN_LIST FEATURES)
     vcpkg_list(APPEND options "--enable-nls")
 else()
-    set(ENV{AUTOPOINT} true) # true, the program
     vcpkg_list(APPEND options "--disable-nls")
 endif()
+set(ENV{AUTOPOINT} true) # true, the program
 
 if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")
     vcpkg_list(APPEND options "CPPFLAGS=\$CPPFLAGS -DIDN2_STATIC")

--- a/ports/libidn2/vcpkg.json
+++ b/ports/libidn2/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "libidn2",
   "version": "2.3.4",
-  "port-version": 2,
+  "port-version": 3,
   "description": "GNU Libidn is an implementation of the Stringprep, Punycode and IDNA 2003 specifications. Libidn's purpose is to encode and decode internationalized domain names.",
   "homepage": "https://www.gnu.org/software/libidn/",
   "license": null,

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -4294,7 +4294,7 @@
     },
     "libidn2": {
       "baseline": "2.3.4",
-      "port-version": 2
+      "port-version": 3
     },
     "libigl": {
       "baseline": "2.4.0",

--- a/versions/l-/libidn2.json
+++ b/versions/l-/libidn2.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "20e169744f242f67bc4b4dc310f5785d55dba58c",
+      "version": "2.3.4",
+      "port-version": 3
+    },
+    {
       "git-tree": "4784d5f7f99d7ea1ebe6f1ef01943e402e7bfddf",
       "version": "2.3.4",
       "port-version": 2

